### ISSUE TITLE
Makefile: FDISK problem

### ISSUE THREAD

**Issue by BTaskaya** [hhuOS/hhuOS#8]
```
ISO image produced: 13087 sectors
Written to medium : 13087 sectors at LBA 0
Writing to 'stdio:build/hhuOS.iso' completed successfully.

Formatting 'hdd1.img', fmt=raw size=33554432
Mapped file hdd1.img as /dev/loop6.
[sudo] password for batuhan: 

Welcome to fdisk (util-linux 2.32).
Changes will remain in memory only, until you decide to write them.
Be careful before using the write command.

The old squashfs signature may remain on the device. It is recommended to wipe the device with wipefs(8) or fdisk --wipe, in order to avoid possible collisions.

Device does not contain a recognized partition table.
Created a new DOS disklabel with disk identifier 0x55da77bd.

Command (m for help): Partition type
   p   primary (0 primary, 0 extended, 4 free)
   e   extended (container for logical partitions)
Select (default p): Partition number (1-4, default 1): First sector (2048-171607, default 2048): Last sector, +sectors or +size{K,M,G,T,P} (2048-171607, default 171607): 
Created a new partition 1 of type 'Linux' and of size 82,8 MiB.

Command (m for help): Selected partition 1
Hex code (type L to list all codes): Changed type of partition 'Linux' to 'unknown'.

Command (m for help): fdisk: failed to write disklabel: Operation not permitted
make: *** [Makefile:116: hdd1.img] Error 1
```

**Comment by faruh100** [comment]
Did you enter the sudo-password correctly? When you buil hhuOS for the first time, you need root-privileges, for mounting the hdd-image.

**Comment by chges100** [comment]
I have this problem too, with Ubuntu 18.04. I have done no further research about it, but entering
the root-pw did not help. Otherwise, the image file hdd1.img is built correctly, I can run the image without problems despite of this error.

**Comment by filkra** [comment]
Fixed with 43702c27fe9769eb8a32db49d6f380c81eff907d
